acceptance/tests/language/pcore_generate_env_isolation.rb

```diff
@@ -0,0 +1,66 @@
+test_name 'C98345: ensure pcore generate assures env. isolation' do
```

Comment:
I don't know what "pcore" is and the Test Rail entry doesn't use that term. Can either "pcore" be changed with "puppet generate", or the Test Rail entry updated with a link to where "pcore" is documented?